session-2/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"time"
+)
+
+const AvailableTask int = 100
+
+// Task priority levels
+const (
+	Low int = iota
+	Medium
+	High
+)
+
+const (
+	Active int = iota
+	InProgress
+	Completed
+)
+
+func main() {
+
+	var projectName string = "Task Management System"
+	var startDate = time.Date(2024, 9, 18, 19, 0, 0, 0, time.UTC)
+
+	fmt.Printf("Welcome to the %s\n", projectName)
+	fmt.Println("Project start date is:", startDate)
+	fmt.Printf("Project: %s \n", projectName)
+
+	/*I know it must be string, I change it when learning function,
+	I write getStatus function and it return string data for status */
+
+	var projectStatus int = InProgress
+	var tasksCompleted int = 25
+
+	fmt.Println("Current project status: ", projectStatus)
```

Comment:
As of result, return should be Status like "In Progress" value with uppercase. projectStatus is int, so is not returning what we expect